editor/plugins/version_control_editor_plugin.cpp

```diff
@@ -141,8 +143,8 @@ void VersionControlEditorPlugin::_initialize_vcs() {
 	if (_load_plugin(selected_plugin)) {
 		_set_up();
 
-		ProjectSettings::get_singleton()->set("editor/version_control/autoload_on_startup", true);
-		ProjectSettings::get_singleton()->set("editor/version_control/plugin_name", selected_plugin);
+		ProjectSettings::get_singleton()->set(SNAME("editor/version_control/autoload_on_startup"), true);
+		ProjectSettings::get_singleton()->set(SNAME("editor/version_control/plugin_name"), selected_plugin);
```

Comment:
This doesn't need to use `SNAME()`. `SNAME()` allocates static memory and should only be used in critical code paths which are used a lot. I assume this one only runs once so it's fine to let it convert the strings to StringName on the fly.